PluginManager.php

```diff
@@ -1,48 +1,145 @@
-<?php
-/*
-* This file is part of EC-CUBE
-*
-* Copyright(c) 2000-2015 LOCKON CO.,LTD. All Rights Reserved.
-* http://www.lockon.co.jp/
-*
-* For the full copyright and license information, please view the LICENSE
-* file that was distributed with this source code.
-*/
-
-namespace Plugin\OrderPdf;
-
-use Eccube\Plugin\AbstractPluginManager;
-
-class PluginManager extends AbstractPluginManager
-{
-
-    public function __construct()
-    {
-    }
-
-    public function install($config, $app)
-    {
-        $this->migrationSchema($app, __DIR__ . '/Migration', $config['code']);
-    }
-
-    public function uninstall($config, $app)
-    {
-        $this->migrationSchema($app, __DIR__ . '/Migration', $config['code'], 0);
-    }
-
-    public function enable($config, $app)
-    {
-
-    }
-
-    public function disable($config, $app)
-    {
-
-    }
-
-    public function update($config, $app)
-    {
-
-    }
-
-}
+<?php
+/*
+ * This file is part of the Order Pdf plugin
+ *
+ * Copyright (C) 2016 LOCKON CO.,LTD. All Rights Reserved.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Plugin\OrderPdf;
+
+use Eccube\Application;
+use Eccube\Plugin\AbstractPluginManager;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Class PluginManager.
+ */
+class PluginManager extends AbstractPluginManager
+{
+    /**
+     * @var string
+     */
+    private $logoName = 'logo.png';
+
+    /**
+     * @var string
+     */
+    private $logoPath;
+
+    /**
+     * PluginManager constructor.
+     */
+    public function __construct()
+    {
+        $this->logoPath = __DIR__.'/Resource/template/'.$this->logoName;
+    }
+
+    /**
+     * Install.
+     *
+     * @param array       $config
+     * @param Application $app
+     */
+    public function install($config, $app)
+    {
+        // Backup logo.
+        $this->backupLogo($app['config']);
+    }
+
+    /**
+     * Uninstall.
+     *
+     * @param array       $config
+     * @param Application $app
+     */
+    public function uninstall($config, $app)
+    {
+        // Remove temp
+        $this->removeLogo($app['config']);
+
+        $this->migrationSchema($app, __DIR__.'/Resource/doctrine/migration', $config['code'], 0);
+    }
+
+    /**
+     * Enable.
+     *
+     * @param array       $config
+     * @param Application $app
+     */
+    public function enable($config, $app)
+    {
+        // Backup logo.
+        $this->backupLogo($app['config']);
```

Comment:
```
public function install($config, $app)
{
    // Backup logo.
    $this->backupLogo($app['config']);
}
```

```
public function enable($config, $app)
{
    // Backup logo.
    $this->backupLogo($app['config']);
    
    $this->migrationSchema($app, __DIR__.'/Resource/doctrine/migration', $config['code']);
}
```

`$this->backupLogo($app['config']);` is duplicated.
i think, its unnecessary in `function enable`.